lib/suma/api/commerce.rb

```diff
@@ -145,11 +160,10 @@ def lookup_editable!
             checkout.payment_instrument = instrument if instrument
           end
 
-          if (fuloptid = params[:fulfillment_option_id])
-            fulopt = checkout.cart.offering.fulfillment_options_dataset[fuloptid]
-            merror!(403, "Fulfillment option not found", code: "resource_not_found") unless fulopt
-            checkout.fulfillment_option = fulopt
-          end
+          fuloptid = params[:fulfillment_option_id]
```

Comment:
Why is this code so different from `modify_fulfillment`?

Reply:
They need to be consistent thanks for catching this, fixing